eng/jacoco-test-coverage/pom.xml

```diff
@@ -464,9 +464,9 @@
       <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-messaging;current} -->
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
+      <groupId>com.azure.spring</groupId>
       <artifactId>spring-cloud-starter-azure-appconfiguration-config</artifactId>
```

Comment:
The artifact id also need to be changed. https://azure.github.io/azure-sdk/java_spring.html#maven